swc-uxp-starter/README.md

```diff
@@ -0,0 +1,168 @@
+# **UXP Plugin with Spectrum Web Component**
+Since UXP v7.0
+# **Objective**
+This sample will help you to create a Spectrum Web Component (SWC) based UXP plugin. It includes setting up the project, integrating the SWC components, and mounting it over the host applications - Photoshop and XD.
+
+Start by learning about [Spectrum Web Components](https://opensource.adobe.com/spectrum-web-components/index.html)
+
+
+# **Getting started**
+1. Start by installing the dependencies `yarn install`
+2. Prepare the bundle using Webpack `yarn build`. You will notice a **dist** folder after this step.
+3. Add the plugin in UXP Developer Tool (UDT). Click on 'Add Plugin' in UDT and navigate to the `manifest.json` inside the dist folder.
+4. Load the plugin in Photoshop/XD by launching the app. Click on the plugin action menu in UDT and select 'Load'.
```

Comment:
We should ask the developers to use the Advanced -> Plugin builder folder option instead of asking them to use the manifest.json from the dist folder.